### ISSUE TITLE
Question about the learning rate

### ISSUE THREAD

**Issue by JiaweiShiCV** [NVlabs/stylegan2-ada-pytorch#129]
I am confused about the learning rate strategy. Does it update every iteration? I have tried to output the learning rate during training, like the following:
``` 
            #  ./training/training_loop.py
            with torch.autograd.profiler.record_function(phase.name + '_opt'):
                for param in phase.module.parameters():
                    if param.grad is not None:
                        misc.nan_to_num(param.grad, nan=0, posinf=1e5, neginf=-1e5, out=param.grad)
                phase.opt.step()
           # output
                if rank == 0:
                    print(phase.opt.param_groups[0]["lr"])
```
But the output  learning rates are always **the same**:
```
G_lr: 0.002
D_lr: 0.0023529411764705885
```
Does the learning rate decay only work once?
```
            # Lazy regularization.
            mb_ratio = reg_interval / (reg_interval + 1)
            opt_kwargs = dnnlib.EasyDict(opt_kwargs)
            opt_kwargs.lr = opt_kwargs.lr * mb_ratio
```

**Comment by zwishenzug** [comment]
The learning rate doesn't change during training.

The Lazy regularization code there isn't a learning rate decay, it's just adjusting the learning rate (once at the start of the program run) to compensate for the lazy regularization interval, so if you have that interval set at 16, it's going to tweak the learning rate by 16/17.
